### ISSUE TITLE
How to change the sample rate?

### ISSUE THREAD

**Issue by luizmanke** [IRNAS/ppk2-api-python#5]
Hi! Nice module!

I've been using the PPK2 and your python module to read the current from nRF52840.
Now I want to change the **sample rate** of PPK2.
How can I do it with your python module?

Here's an image of how it's done using the Power Profiler from nRF Connect v3.6.1 :

![image](https://user-images.githubusercontent.com/37818079/111360182-34c6d280-866b-11eb-9396-360daaef61c6.png)

**Comment by NejcKle** [comment]
Hi @luizmanke! 

I'm glad to hear you've been using this module! 
As far as we've discovered the sample rate **cannot** be changed and is set at 100k samples per second. 
The setting in the nRF Connect App appears to be averaging in a window for prettier data display. 

We've implemented something similar here: https://github.com/IRNAS/ppk2-api-python/blob/ab39f6baaaf4d9af3c554611a14aac6d643d4602/power_profiler.py#L119-L126
